scriptoocloud/src/main/java/edu/asu/diging/scriptoocloud/web/EditProjectController.java

```diff
@@ -0,0 +1,52 @@
+package edu.asu.diging.scriptoocloud.web;
+
+import java.security.Principal;
+import java.util.Map;
+import java.util.Optional;
+
+import javax.validation.Valid;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.validation.BindingResult;
+import org.springframework.web.bind.annotation.ModelAttribute;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.servlet.mvc.support.RedirectAttributes;
+
+import edu.asu.diging.scriptoocloud.core.data.ProjectRepository;
+import edu.asu.diging.scriptoocloud.core.model.Project;
+import edu.asu.diging.scriptoocloud.core.model.impl.ProjectImpl;
+import edu.asu.diging.scriptoocloud.core.service.ProjectManager;
+
+@Controller
+public class EditProjectController {
+
+    @Autowired
+    private ProjectManager projectManager;
+    
+    @RequestMapping(value = "/auth/edit/{id}", method = RequestMethod.GET)
+    public String get(@Valid @ModelAttribute("project") ProjectImpl projectImpl, BindingResult result,
```

Comment:
`projectImpl` shouldn't be passed as argument if it is not needed (which it doesn't seem to since you're just replacing its value in line 35 anyways)

Reply:
Removed